sdk/synapse/Azure.Analytics.Synapse.Spark/api/Azure.Analytics.Synapse.Spark.netstandard2.0.cs

```diff
@@ -160,11 +160,13 @@ public partial class SparkBatchJobCollection
     public partial class SparkBatchJobOptions
     {
         public SparkBatchJobOptions(string name, string file) { }
+        public SparkBatchJobOptions(string name, string file, Azure.Analytics.Synapse.Spark.Models.SparkBatchOperationCompletionType creationCompletionType) { }
```

Comment:
Would it make sense to set the default value in the constructor so it's clear to the caller what will happen if they don't pass the value?

Reply:
It already has a default value: https://github.com/idear1203/azure-sdk-for-net/blob/synapse_fix_polling_logic_for_spark/sdk/synapse/Azure.Analytics.Synapse.Spark/src/Customization/Models/SparkBatchJobOptions.cs#L26 . I will run the export API script again to see if the default value can be also shown here.

Reply:
I ran the `Export-API.ps1` again but nothing changed.

Reply:
I believe you will need to move where you set the default value from the property into the constructor.

Reply:
I see. Will do.